scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SMALL_INSTALL=false
+
+case "${1:-}" in
+    -h|--help)
+        echo -e "Usage:\n\tquick-helm-install.sh [options]"
+        echo " "
+        echo "Installs StackRox via Helm charts, requires Helm."
+        echo " "
+        echo "options:"
+        echo "-h, --help                show brief help"
+        echo "-s, --small               reduce StackRox resource requirements for small clusters"
+        exit 0
+        ;;
+    -s|--small)
+        SMALL_INSTALL=true
+        ;;
+esac
+
+echo "Adding the stackrox/helm-charts/opensource repository to Helm."
+
+helm repo add stackrox https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
+
+echo "Generating STACKROX_ADMIN_PASSWORD"
+
+STACKROX_ADMIN_PASSWORD="$(openssl rand -base64 20 | tr -d '/=+')"
+
+echo "Installing stackrox-central-services"
+
+helm install -n stackrox --create-namespace stackrox-central-services stackrox/stackrox-central-services --set central.adminPassword.value="${STACKROX_ADMIN_PASSWORD}"
+
+if $SMALL_INSTALL; then
+    echo "Upgrading stackrox to a small install"
+    helm upgrade -n stackrox stackrox-central-services stackrox/stackrox-central-services \
+      --set central.resources.requests.memory=1Gi \
+      --set central.resources.requests.cpu=1 \
+      --set central.resources.limits.memory=4Gi \
+      --set central.resources.limits.cpu=1 \
+      --set scanner.autoscaling.disable=true \
+      --set scanner.replicas=1 \
+      --set scanner.resources.requests.memory=500Mi \
+      --set scanner.resources.requests.cpu=500m \
+      --set scanner.resources.limits.memory=2500Mi \
+      --set scanner.resources.limits.cpu=2000m
+fi
+
+kubectl -n stackrox rollout status deploy/central --timeout=3m
+
+echo "Forwarding ports to central"
+
+kubectl -n stackrox port-forward deploy/central --pod-running-timeout=1m0s 8000:8443 > /dev/null 2>&1 &
+
+echo "Generating an init bundle with shared secrets"
+
+kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
+  --password "${STACKROX_ADMIN_PASSWORD}" \
+  central init-bundles generate stackrox-init-bundle --output - 1> stackrox-init-bundle.yaml
+
+installflags=("-f stackrox-init-bundle.yaml" "--set clusterName=\"my-secured-cluster\"")
+if $SMALL_INSTALL; then
+    installflags+=("--set sensor.resources.requests.memory=500Mi")
+    installflags+=("--set sensor.resources.requests.cpu=500m")
+    installflags+=("--set sensor.resources.limits.memory=500Mi")
+    installflags+=("--set sensor.resources.limits.cpu=500m")
+    echo "Installing the first small secured cluster"
+else
+    echo "Installing the first secured cluster"
+fi
+
+helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services ${installflags[*]}
```

Comment:
Now I see why it worked :)

We should not use `${installflags[*]}`, instead, I suggest `"${installflags[@]}" (with quotes):

 ```suggestion
helm install -n stackrox stackrox-secured-cluster-services stackrox/stackrox-secured-cluster-services "${installflags[@]}"
```

Reply:
Why is this better?

Reply:
Is mainly about handling spaces and setting boundaries between flags and their values. Compare:

```shell
ARR+=("-f file with spaces.yaml")
some_command ${ARR[*]}
```

```shell
ARR+=("-f" "file with spaces.yaml")
some_command "${ARR[@]}"
```

You could also think about such cases (untested and maybe better not to test it 😄 )
```shell
ARR+=("-f file with spaces.yaml; rm -fr /your-file-system")
some_command ${ARR[*]}
```

Reply:
ShellCheck should warn you about this, could you please make sure that you have it enabled?
I am not saying you should listen to `shellcheck` all the time, but in 90% of cases the suggestions are justified.